stock_release_channel_delivery/migrations/16.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if openupgrade.column_exists(env.cr, "stock_release_channel", "carrier_id"):
+        openupgrade.rename_columns(
+            env.cr, {"stock_release_channel": [("carrier_id", None)]}
```

Comment:
why None?

Reply:
Hi anh, as I read here: https://github.com/OCA/openupgradelib/blob/d341a6a9e72a051c23ceefe622040c456f37767a/openupgradelib/openupgrade.py#L609

Since I use  `get_legacy_name` in post-migration.py, use None to trigger it
Please help to correct If I'm wrong

Reply:
ah ok, good for traceability as if I understand correctly the column will be kept even after upgrade is complete
oca guys might want to get rid of it and feed carrier_ids directly frtom carrier_id, but let's see
ok for me